### DISCUSSION TITLE
helloworld.py error

### DISCUSSION THREAD

**Discussion by tturkmen** [USC-ACTLab/crazyswarm#453] [General]
![2021-08-07 19-12-58 ekran görüntüsü](https://user-images.githubusercontent.com/4279243/128606658-1315a009-035e-461a-9137-709188f353f9.png)
Thanks for your works
Hello I have crazyflie 2.1 with flow deck, connect from crazyradio Pa
 I installed physcial and robot simulations but when run helloworld.py with roslaunch, have srv error. How can solve this problem.

**Comment by whoenig** [comment]
You need to source your ROS workspace in the second terminal as well.

**Comment by tturkmen** [comment]
When I run pytest
![2021-08-08 07-00-06 ekran görüntüsü](https://user-images.githubusercontent.com/4279243/128620254-eef7e44f-7236-42a0-8212-06a1ca3e4201.png)
Then source and run helloworld.py with source
![2021-08-08 05-55-46 ekran görüntüsü](https://user-images.githubusercontent.com/4279243/128619231-e3561d90-b64b-4fe0-8df0-9ed402ff957c.png)
I source like photo but problem not solved yet.
But only run sim mode no problem.

**Reply by whoenig** [reply]
Perhaps you pulled the latest Crazyswarm changes, but did not rebuilt using `build.sh`? This sounds like the ROS package is not properly built.

**Reply by tturkmen** [reply]
Yes I pulled changes and use build.sh. But I was ubuntu 18.04. Now I use ubuntu 20.04 and problem solved. Thanks..
